modules/govuk_jenkins/templates/jobs/copy_data_to_integration.yaml.erb

```diff
@@ -60,6 +70,11 @@
             HOSTNAME=deploy.integration.publishing.service.gov.uk
             API_KEY=<%= @ci_alphagov_api_key %>
             AUTH_TOKEN=<%= @auth_token %>
+      - slack:
+          team-domain: <%= @slack_team_domain %>
+          auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
```

Comment:
@surminus we have also removed the environment variable from integration and staging in order to reduce the number of slack notifications in the channel. If you would like to check this in integration we will need to re-add the same environment variable in https://github.gds/gds/deployment/commit/fe0805b337bbcf9e7d6817e3e4497047e8286ba1 to the integration file.